content/general/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Data Types'
+Description: 'Data is classified into data types that tell the compiler how the data is intended to be used.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/build-web-apps-with-asp-net'
+---
+
+Data is the foundation of all programs. In programming, data is classified into different types to let the compiler know how the data can be used.
```

Comment:
```suggestion
Data is foundational in programming and is classified into different types to indicate how it can be interpreted and used in the compiler.
```